carbon-design-system/carbon-addons-iot-react
# fix(sidepanel): not to show tooltip when closed inline

Closes #3695

**Summary**

- not able to tab on content when side panel is closed

**Change List (commits, features, bugs, etc)**

- SidePanel.jsx  added `tabIndex` and `aria-hidden`

**Acceptance Test (how to verify the PR)**

- go to side panel [story](https://deploy-preview-3744--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-sidepanel--default)
- uncheck isOpen
- choose `inline`
- click on frame and tab 
- see any tooltip or any content showed up or focused

**Regression Test (how to make sure this PR doesn't break old functionality)**

- tests here



**Things to look for during review**
